crates/bevy_window/src/window.rs

```diff
@@ -631,10 +631,10 @@ impl Default for WindowResolution {
 
 impl WindowResolution {
     /// Creates a new [`WindowResolution`].
-    pub fn new(logical_width: f32, logical_height: f32) -> Self {
+    pub fn new(logical_size: LogicalSize) -> Self {
         Self {
-            physical_width: logical_width as u32,
-            physical_height: logical_height as u32,
+            physical_width: logical_size.x as u32,
+            physical_height: logical_size.y as u32,
```

Comment:
Its  quite deceptive and doesn't really make sense for the extents of `logical_size` to be floats when they are converted into f32s immediately.